### DISCUSSION TITLE
How to solve ./gradlew build 30% successful test

### DISCUSSION THREAD

**Discussion by rich7420** [apache/gravitino#3329] [Q&A] [answered]
I follow [this](https://datastrato.ai/docs/0.5.0/how-to-build/) to build.
When I use ./ gradlew build,the error message comes.
Can I ask why it has appeared so many failed test?
![image](https://github.com/datastrato/gravitino/assets/101171023/79ffe45d-fb45-4fbc-b7c2-adafb387c332)
![image](https://github.com/datastrato/gravitino/assets/101171023/25d36d93-6f67-496b-962d-1eece489f12b)
Is my environmental issue the cause of this problem?
I would be very grateful if someone helps me.

**Comment by yuqi1129** [comment]
Can you please provide the detailed error information here? From the message you provided, I can't tell what's wrong here.

**Reply by rich7420** [reply]
Ok! 
there is wrong message I got.
every wrong test have report like this.
![image](https://github.com/datastrato/gravitino/assets/101171023/54f6820f-e8d3-4585-88e2-856d3ef5f355)

**Reply by rich7420** [reply]
Thank you for your answer.
Is there any way to find helpful error messages?
![image](https://github.com/datastrato/gravitino/assets/101171023/a580c742-05dc-40d2-929c-edbbb3297c0b)

**Comment by yuqi1129** [comment]
Can you provide the following information here? like OS, JDK

**Reply by rich7420** [reply]
Sure. My OS is Ubuntu22.04.4 and there is my JDK below
![image](https://github.com/datastrato/gravitino/assets/101171023/7a437929-e49c-40fb-8740-fc06c82d8aa3)

**Comment by mchades** [comment, marked as answer]
It's because the docker server doesn't start, see #3336
